test/elasticsearch_reader/fetcher-spec.ts

```diff
@@ -137,6 +233,28 @@ describe('elasticsearch_reader fetcher', () => {
         expect(recordCount).toEqual(evenSpread.data.length);
     });
 
+    it('fetcher throws if query size exceeds the index.max_result_window setting', async () => {
+        // this range has 48 records
+        const slice = {
+            start: '2019-04-26T15:00:23.201Z',
+            end: '2019-04-26T15:00:23.220Z',
+            limit: '2019-04-26T15:00:23.394Z',
```

Comment:
Oh, this `limit` is copied here from other examples, I don't even know what it does.  Maybe Jared does.